tests/static-checks.sh

```diff
@@ -13,14 +13,14 @@ set -e
 [ -n "$DEBUG" ] && set -x
 
 cidir=$(realpath $(dirname "$0"))
-source "${cidir}/lib.sh"
+source "${cidir}/common.bash"
 
 # By default in Golang >= 1.16 GO111MODULE is set to "on",
 # some subprojects in this repo may not support "go modules",
 # set GO111MODULE to "auto" to enable module-aware mode only when
 # a go.mod file is present in the current directory.
 export GO111MODULE="auto"
-export tests_repo="${tests_repo:-github.com/kata-containers/tests}"
+export tests_repo="${tests_repo:-github.com/kata-containers/kata-containers/tests}"
```

Comment:
I would drop "repo" from the variable names as this is not technically a repository anymore and use variable names just as `test_path` and `test_dir` instead.

Reply:
@amshinde updated